src/SprykerEco/Zed/Unzer/Persistence/UnzerRepository.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Persistence;
+
+use Generated\Shared\Transfer\MerchantUnzerParticipantTransfer;
+use Generated\Shared\Transfer\PaymentUnzerOrderItemCollectionTransfer;
+use Generated\Shared\Transfer\PaymentUnzerOrderItemTransfer;
+use Generated\Shared\Transfer\PaymentUnzerTransactionTransfer;
+use Generated\Shared\Transfer\PaymentUnzerTransfer;
+use Spryker\Zed\Kernel\Persistence\AbstractRepository;
+use SprykerEco\Zed\Unzer\Persistence\Mapper\UnzerPersistenceMapper;
+
+/**
+ * @method \SprykerEco\Zed\Unzer\Persistence\UnzerPersistenceFactory getFactory()
+ */
+class UnzerRepository extends AbstractRepository implements UnzerRepositoryInterface
```

Comment:
Please, check this RFC https://spryker.atlassian.net/wiki/spaces/RFC/pages/2594832674?atlOrigin=eyJpIjoiODg4MDJmMGFkZjYyNDE2YmExZDYyMTk5YjhmZjEyM2YiLCJwIjoiY29uZmx1ZW5jZS1jaGF0cy1pbnQifQ

It is possible that you'll need to some changes to your methods according to it.